File: Php/app/Cashgame/CashgameNavigationModel.php
namespace app\Cashgame{

	use entities\Cashgame;
	use entities\Homegame;

	class CashgameNavigationModel{

		public $pageNavModel;
		public $yearNavModel;

		public function __construct(Homegame $homegame, $view, $years, $year = null, Cashgame $runningGame = null){
			pageNavModel = new CashgamePageNavigationModel($homegame, $year, $view, $runningGame);
			yearNavModel = new CashgameYearNavigationModel($homegame, $years, $year, $view);
		}

	}

}

File: Web/Models/NavigationModels/CashgameNavigationModel.cs
using System.Collections.Generic;
using Core.Classes;

namespace Web.Models.NavigationModels{
    public class CashgameNavigationModel{

        public CashgamePageNavigationModel PageNavModel { get; set; }
        public CashgameYearNavigationModel YearNavModel { get; set; }

        public CashgameNavigationModel(Homegame homegame, string view, List<int> years, int? year, Cashgame runningGame){
			PageNavModel = new CashgamePageNavigationModel(homegame, year, view, runningGame);
			YearNavModel = new CashgameYearNavigationModel(homegame, years, year, view);
		}
    
    }

}
